src/content/homepage.md

```diff
@@ -203,30 +203,84 @@ milestonesSection:
         text: First dApp PoC
         when: 'Jan 2022'
         isMajor: true
-      - order: 1
+      - order: 2
         text: Desktop wallet upgrade
         when: 'Apr 2022'
         isMajor: false
-      - order: 2
+      - order: 3
         text: Explorer optimization & addition of statistics
         when: 'May 2022'
         isMajor: false
-      - order: 3
+      - order: 4
         text: Full node 1.4.0 with all of the major features for the Leman network upgrade implemented
         when: 'Jun 2022'
         isMajor: false
-      - order: 4
+      - order: 5
         text: Beta version of contract SDK (alephium-web3)
         when: 'Jun 2022'
         isMajor: false
-      - order: 5
+      - order: 6
         text: New testnet launched
         when: 'Jun 2022'
         isMajor: false
-      - order: 6
+      - order: 7
         text: Leman Network Upgrade deployed on the testnet
         when: 'Jun 2022'
         isMajor: true
+      - order: 8
+        text: Improve documentation of the core infrastructure
+        when: 'H2 2022'
+      - order: 9
+        text: Improve the Alephium SDK with more features
+        when: 'H2 2022'
+      - order: 10
+        text: Improve the robustness and efficiency of the full node &  explorer backend
+        when: 'H2 2022'
+      - order: 11
+        text: Alpha version of multi-guardians bridge on the Testnet
+        when: 'Nov 2022'
+      - order: 12
+        text: NFT prototype alpha released
+        when: 'Q4 2022'
+      - order: 13
+        text: Improved DEX prototype
+        when: 'Q4 2022'
+    - year: 2023
+      entries:
+      - order: 0
+        text: Launch Browser Wallet Extension
+        when: 'Mar 2023'
+      - order: 1
+        text: Leman Network Upgrade on Mainnet
+        when: '27.03.2023'
+        isMajor: true
+      - order: 2
+        text: Cross-chain interoperability
+        when: 'Mar 2023'
+      - order: 3
+        text: Schnorr signatures
+        when: 'Mar 2023'
+      - order: 4
+        text: New set of Virtual Machine (VM) instructions and building functions
+        when: 'Mar 2023'
+      - order: 5
+        text: New Asset Permission System
+        when: 'Mar 2023'
+      - order: 6
+        text: External calls Check System
+        when: 'Mar 2023'
+      - order: 7
+        text: Simpler UTXO management
+        when: 'Mar 2023'
+      - order: 8
+        text:  Improved node APIs and SDK
+        when: 'Mar 2023'
+      - order: 9
+        text:  Improved difficulty adjustment algorithm (DAA)
+        when: 'Mar 2023'
+      - order: 10
+        text:  Improved contract storage for security and performance
+        when: 'Mar 2023'
```

Comment:
Those 9 items are all attached to March 2023. Maybe there's a way to merge some of them?